example/rater/generated_answer.ipynb

```diff
@@ -0,0 +1,355 @@
+     ]
+    }
+   ],
+   "source": [
+    "config = RaterForGeneratedAnswerConfig(\n",
+    "    flow_name=\"RaterFlow\",\n",
+    "    model_config=OpenAIModelConfig(num_call=5, response_format={\"type\": \"text\"}),\n",
+    "    label2score={\n",
+    "        \"Strong accept\": 2.0,\n",
+    "        \"Accept\": 1.0,\n",
+    "        \"Equivalent\": 0.0,\n",
+    "        \"Reject\": -1.0,\n",
+    "        \"Strong reject\": -2.0,\n",
+    "    }\n",
+    ")\n",
+    "client = RaterClient(config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we can run this client. Client will sample output from LLM 5 times to generate final result."
```

Comment:
Let's add a header here named "Run the client". 

Also, we can add more details such as "Then we can run the client. For each item in the `raw_input`, the Client will generate an explanation and a final label [`Strong accept`, `Accept`, `Equivalent`, `Reject`,`Strong reject`] . The label is decided by taking the majority votes from sampling the LLM output 5 times, which improved stability compared with outputting 1 time.